site/src/pages/release-notes/1.14.0.mdx

```diff
@@ -0,0 +1,337 @@
+---
+date: 2022-1-27
+---
+
+## 🌟 New features
+
+- You can now use [Scala 3](https://dotty.epfl.ch/) for `armeria-scala` and `armeria-scalapb`. #3614 #4036
+  <Tip>
+
+  Scala 3 modules are currently experimental. We're looking forward to your feedback.
+
+  </Tip>
+
+- You can now fluently convert an <type://HttpResponse> into another using <type://WebClient>. #4021
+  ```java
+  WebClient client = WebClient.of("https://api.example.com");
+  CompletableFuture<ResponseEntity<MyObject>> response =
+      client.prepare()
+            .get("/v1/items/1")
+            .asJson(MyObject.class)
+            .execute();
+  ``` 
+- You can now use <type://BlockingWebClient> to wait for an <type://HttpResponse> to be completed. #4021
+  ```java
+  BlockingWebClient client = BlockingWebClient.of("https://api.example.com");
+  ResponseEntity<MyObject> response =
+      client.prepare()
+            .get("/v1/items/1")
+            .asJson(MyObject.class)
+            .execute();
+  ``` 
+  <Tip>
+
+  Note that you should never use this client in an `EventLoop` thread.
+  Use it from a non-`EventLoop` thread such as <type://BlockingTaskExecutor>.
+ 
+  </Tip>
+- You can now fluently build a gRPC client with various options using
+  <type://GrpcClientBuilder>. #3981 #3999 #3964 #3975
+  ```java
+  GrpcClients.builder(...)
+             .decorator(myDecorators)
+             .maxResponseMessageLength(MAX_MESSAGE_SIZE)
+             .jsonMarshallerFactory(descriptor -> {
+                 ...
+             })
+             .intercept(myInterceptors)
+             .build(MyStub.class);
+  ``` 
+- You can now fluently build an <type://HttpResponse> by using <type://HttpRequestBuilder>. #3398 #3941
+  ```java
+  HttpResponse.builder()
+              .ok()
+              .header(HttpHeaderNames.USER_AGENT, "Armeria")
+              .content(content)
+              .build(); 
+  ```
+- You can now use <type://StreamMessage#mapAsync(Function)> to transform a <type://StreamMessage> 
+  using an async operation. #3916 #3962
+  ```java
+  StreamMessage<Integer> userIds = StreamMessage.of(...);
+  StreamMessage<UserInfo> userInfos = streamMessage.mapAsync(this::getUserInfo);
+  
+  CompletableFuture<UserInfo> getUserInfo(int userId) {
+      ...
+  }
+  ```
+- You can now use <type://HttpMessage#peekData(Consumer)>, <type://HttpMessage#peekTrailers(Consumer)> and
+  <type://StreamMessage#peekError(Consumer)> and <type://HttpResponse#peekHeaders(Consumer)> operators on
+  <type://HttpRequest> and <type://HttpResponse>. #3097 #3988 #3949
```

Comment:
when listing multiple subjects, "and" should link only the last subjects

```suggestion
- You can now use <type://HttpMessage#peekData(Consumer)>, <type://HttpMessage#peekTrailers(Consumer)>,
  <type://StreamMessage#peekError(Consumer)> and <type://HttpResponse#peekHeaders(Consumer)> operators on
  <type://HttpRequest> and <type://HttpResponse>. #3097 #3988 #3949
```

Also, it might be easier on the eye if the order were header -> data -> trailer -> error instead.

Reply:
Good suggestion!